modules/t/vcf_tabix.t

```diff
@@ -115,6 +115,16 @@ ok($parser->get_metadata_description('INFO', 'DP') eq 'Total Depth', 'getMetaDes
 
 ok ($parser->close(), "Closing file");
 
+note "\n> Testing reading bgz files:\n";
+my $test_file2 = $FindBin::Bin . '/input/data.vcf.bgz';
+my $parser2 = Bio::EnsEMBL::IO::Parser::VCF4Tabix->open($test_file2);
```

Comment:
Could we test triggering that error by providing an unsupported file?

Reply:
Done.